cli/azd/pkg/auth/manager.go

```diff
@@ -263,10 +270,28 @@ func newCredentialFromClientAssertion(
 	return cred, nil
 }
 
-func (m *Manager) LoginInteractive(ctx context.Context, redirectPort int) (azcore.TokenCredential, error) {
+type LoginInteractiveOptions struct {
+	RedirectPort int
+	TenantId     string
+}
+
+func (m *Manager) LoginInteractive(
+	ctx context.Context, loginOptions *LoginInteractiveOptions) (azcore.TokenCredential, error) {
+
+	// tenantId allow to set an specific tenant id for doing log in, when this is set, the client is re-built using
```

Comment:
I could see this comment being useful as documentation of `LoginInteractiveOptions.TenantId`

Reply:
Agreed - we should either apply some sort of builder pattern or lazily construct the client after all options are available.